website/static/js/quickProjectSearchPlugin.js

```diff
@@ -37,20 +37,19 @@ var quickSearchProject = {
                 self.nodes().push(node);
                 self.retrieveContributors(node);
             });
-            self.populateEligibleNodes(0, self.countDisplayed() - 1);
```

Comment:
It was a result of a comment made by @caneruguz on this PR https://github.com/pattisdr/osf.io/pull/2#discussion-diff-52055508.

He was referencing New and Noteworthy, but since it was a comment on this PR, I figured I'd make the same changes to QuickSearch here.  I assumed he didn't just mean this one instance.